app/src/main/java/ch/admin/bag/dp3t/storage/SecureStorage.java

```diff
@@ -144,20 +146,15 @@ public void setLastShownContactId(int contactId) {
 	}
 
 	public boolean isHotlineCallPending() {
```

Comment:
I think this should be refactored to isLeitfadenOpenPending() ? And also at the places where this function is called there is some renaming necessary.